.github/workflows/ci.yml

```diff
@@ -4,11 +4,12 @@ on: [push]
 jobs:
   tests:
     name: Tests
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        version: ['7.1', 7.2', '7.3', '7.4', '8.0']
+        version: ['7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
```

Comment:
Do we need to keep unsupported versions in the tests?

Reply:
No, I don't think so.

Fixed in b6dbc6a